app/policies/manage/admin/user_policy.rb

```diff
@@ -40,7 +40,7 @@ def allowed_access_levels
   end
 
   def allowed_permissions
-    Permissions::ALL_PERMISSIONS.slice(*user_permissions).values
+    Permissions::ALL_PERMISSIONS.values
```

Comment:
Full review pending, but does this let users assign permissions to themselves/their peers that they shouldn't be allowed to? Eg. can a CVHO give themselves access to Flipper now? It depends on: Who all are allowed to manage users?

Reply:
Good question - I don't know who all has the permissions to 'manage admins'. I guess I assume if someone has that ability, they are a super-admin or something along those lines?  Could someone w/ production access run a query to see how many people have access to manage admins?

Reply:
According to our [permissions file](https://github.com/simpledotorg/simple-server/blob/master/app/policies/permissions.rb#L1), org admins, CVHOs, _and_ superadmins can do this, so it extends well beyond just us.